builtInFunctions/esdtNFTCreate.go

```diff
@@ -331,6 +345,10 @@ func (e *esdtNFTCreate) getESDTNFTCreateInput(
 	var nonce uint64
 	var originalCreator []byte
 	var err error
+	quantity := big.NewInt(0).SetBytes(vmInput.Arguments[1])
```

Comment:
This check was just moved from main ProcessBuiltinFunction

Reply:
it is still not backward compatible. in case someone creates an NFT with 0 quantity and bad royalties, with old code the error message is bad royalties, with the new code it is invalid quantity.